cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -959,14 +959,35 @@ def _add_head_node(self):
                 else None,
                 block_device_mappings=block_device_mappings,
                 key_name=head_node.ssh.key_name,
+                network_interfaces=head_lt_nw_interfaces,
+                image_id=self._cluster_config.ami_id,
+                ebs_optimized=head_node.is_ebs_optimized,
+                iam_instance_profile=CfnLaunchTemplate.IamInstanceProfileProperty(name=self.root_instance_profile.ref),
+                user_data=Fn.base64(
+                    Fn.sub(
+                        head_node_lt_user_data,
+                        {
+                            "YumProxy": head_node.networking.proxy if head_node.networking.proxy else "_none_",
+                            "DnfProxy": head_node.networking.proxy if head_node.networking.proxy else "",
+                            "AptProxy": head_node.networking.proxy if head_node.networking.proxy else "false",
+                            "ProxyServer": head_node.networking.proxy if head_node.networking.proxy else "NONE",
+                            "CustomChefCookbook": self._custom_chef_cookbook(),
```

Comment:
check `_custom_chef_cookbook` func, it seems returning a cookbook object, not a cookbook url.

Reply:
Yep, should be `self._custom_chef_cookbook().chef_cookbook`

Reply:
Thanks!